pkg/kubelet/image_manager.go

```diff
@@ -302,7 +302,10 @@ func (im *realImageManager) freeSpace(bytesToFree int64, freeTime time.Time) (in
 		}
 	}
 
-	return spaceFreed, lastErr
+	if len(deletionErrors) > 0 {
+		return spaceFreed, fmt.Errorf("Wated to free %d, but freed %d space with errors in image deletion: %v", bytesToFree, spaceFreed, deletionErrors)
```

Comment:
nit: s/Wated/wanted

typo and also error message starts with a lower case letter.